src/components/Contacts/ContactFlow/ContactFlowRow/ContactFlowRow.tsx

```diff
@@ -64,13 +64,11 @@ export const StyledAvatar = styled(Avatar)(() => ({
 
 export interface DraggedContact {
   id: string;
-  status: {
-    __typename?: 'IdValue' | undefined;
-  } & Pick<IdValue, 'id' | 'value'>;
+  status: StatusEnum;
   name: string;
   starred: boolean;
-  width: number;
-  contactPhase?: PhaseEnum | null;
+  width: number | undefined;
```

Comment:
Couldn't this be this?
```suggestion
  width?: number;
```

Reply:
My goal here was to make sure that the code adds the width property and doesn't accidentally omit it like it was doing with `contactPhase`. `width?: number` means the you can omit the property. `width: number | undefined` means that you have to set the property, but its value can be `undefined`. Feel free to push back if you still want me to change it.